src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -86,10 +87,36 @@ const initialFormState = {
     paymentBackend: '',
     paymentBackendOptions: {},
     submissionsRemovalOptions: {},
-    confirmationEmailTemplate: null,
+    confirmationEmailTemplate: {
+      translations: {
+        nl: {},
+        en: {},
+      },
+    },
     confirmationEmailOption: 'global_email',
     explanationTemplate: '',
     autoLoginAuthenticationBackend: '',
+    // TODO dynamic
+    translations: {
+      en: {
```

Comment:
Currently the languages and keys are hardcoded, I'm not sure if this is an issue that could be addressed later of or if it needs to be tackled in this PR